.github/CONTRIBUTING.md

```diff
@@ -75,6 +75,22 @@ As a general rule, if a repository A is a submodule dependency of repository B t
 
 - [concordium-reference-wallet-ios](https://github.com/Concordium/concordium-reference-wallet-ios)
   This is a reference implementation of a mobile wallet for the iOS platform.
+  
+- [concordium-desktop-wallet](https://github.com/Concordium/concordium-desktop-wallet)
+  This is an implementation of a desktop wallet for Windows, macOS and Linux.
+  
+    This repository has the following submodule dependencies.
+    - [concordium-base](https://github.com/Concordium/concordium-base)
+    - [concordium-grpc-api](https://github.com/Concordium/concordium-grpc-api)
+  
+- [concordium-ledger-app](https://github.com/Concordium/concordium-ledger-app)
+  This repository contains an application for the Ledger Nano S and Ledger Nano X devices for retrieving keys and signing transactions.
```

Comment:
Do we want to list the external submodule dependencies here as well, or do we only do it for the internal ones?

Reply:
Not sure. We don't have this with other repositories.

But I think since this describes the layout of our repositories let's just list ours.